### ISSUE TITLE
LNDg website error on version v1.9.0rc5

### ISSUE THREAD

**Issue by FlorianHubl** [raspiblitz/raspiblitz#3845]
When I install LNDg on the Raspiblitz and then open it, it shows the address where I can access LNDg. But the browser can't open this website. It says that the site is unavailable.

**Comment by rootzoll** [comment]
@openoms can you check why the virtual env does get created anymore when running the install script?

```
Reading state information... Done
virtualenv is already the newest version (20.4.0+ds-2+deb11u1).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
ModuleNotFoundError: No module named 'virtualenv.activation.xonsh'
sudo: .venv/bin/pip: command not found
sudo: .venv/bin/python: command not found
sudo: .venv/bin/python: command not found
Database already exists, using existing database
```

**Comment by openoms** [comment]
It has to with poetry, unfortunately it seems to interfere with pip.
What can be done is to try.
`sudo pip uninstall virtualenv`
and use the version installed with apt.

Otherwise we will need not to install pip and poetry packages as root (with sudo).

**Comment by rootzoll** [comment]
I run into the same error when I try to install specter on rc5fatpack.

To make the installs of lndg & specter work again - I need todo before runnig them:
```
sudo apt-get remove -y python3-virtualenv
sudo pip uninstall -y virtualenv
sudo apt-get install -y python3-virtualenv
```

**Comment by rootzoll** [comment]
Sideobservation: When installing a v1.9.0rc5-minimal image - the install script of lndg is running out of the box. So it seems that some App from the fatpack (api or one of the bonus apps) is changing the building environment and is breaking the install scripts.

**Comment by rootzoll** [comment]
OK should now be fixed for final v1.9.0
